src/widgets/Tabs/VaccineSupplementalData.js

```diff
@@ -24,11 +24,17 @@ import { PageButtonWithOnePress } from '../PageButtonWithOnePress';
 import { VaccinePrescriptionActions } from '../../actions/Entities/index';
 import { WizardActions } from '../../actions/WizardActions';
 import { Paper } from '../Paper';
+import { selectLastSupplementalData } from '../../selectors/Entities/vaccinePrescription';
 
 const { pageTopViewContainer } = globalStyles;
 
 const VaccineSupplementalDataComponent = ({ onCancel, onComplete, siteSchema }) => {
-  const [{ formData, isValid }, setForm] = useState({ formData: null, isValid: false });
+  const lastSupplementalData = selectLastSupplementalData();
+  const [{ formData, isValid }, setForm] = useState({
+    formData: JSON.parse(lastSupplementalData),
+    isValid: lastSupplementalData ?? false,
```

Comment:
This is probably going to do weird stuff if the form schema is changed in mSupply in between transactions 🤔 . Maybe need to find a way to make the `JSONForm` validate the initial formData

Reply:
Cool - have adjusted the schema fetching to sort by version and pull latest (affects the PCD form as well) 1133a64

The validation of seeded data was hidden in 7dc0ad8 (sorry I should have broken it into something smaller but I got a bit hack happy)